cmd/koordlet/main.go

```diff
@@ -46,6 +46,15 @@ func main() {
 	go wait.Forever(klog.Flush, 5*time.Second)
 	defer klog.Flush()
 
+	if *options.EnablePprof {
+		go func() {
+			klog.V(4).Infof("Starting pprof on %v", *options.PprofAddr)
+			if err := http.ListenAndServe(*options.PprofAddr, nil); err != nil {
```

Comment:
You sure this will provide pprof server without `_ "net/http/pprof"` imported?

Reply:
it happens to works because other related pkg imports this